drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -179,33 +222,98 @@ def load_definition(form_definition:, identifier:, role:)
         role: role,
         version: 0,
         status: 'draft',
-      ).first_or_create!
+      ).first_or_create!(title: title || role.to_s.humanize)
       record.definition = form_definition
+      record.title = title if title.present?
       record.save!
     end
 
-    # Load form definitions for editing and creating records
-    public def seed_record_form_definitions
-      record_forms.merge(service_forms).each do |identifier, form_definition|
+    public def ensure_system_instances_exist!
+      [
+        :project,
+        :organization,
+        :project_coc,
+        :funder,
+        :inventory,
+        :client,
+        :new_client_enrollment,
+        :enrollment,
+      ].each do |identifier|
```

Comment:
Does it make any sense to loop over `Hmis::Form::Definition::SYSTEM_FORM_ROLES` instead of having this list and then comparing it and raising if it doesn't exist?

Reply:
Yes probably, will update